### PR TITLE
feat(wallet-dashboard): style stake overview

### LINKED ISSUE #3695: Style staking overview
- Show CTA if there are no stakes (depends on https://github.com/iotaledger/iota/issues/3685) Image
- Add overview if there are stakes Image

### PR DESCRIPTION
**Pull request by brancoder**
# Description of change

nested PR :warning: 
resolves https://github.com/iotaledger/iota/issues/3695

### CODE REVIEW

File: apps/wallet-dashboard/components/staking-overview/StakingData.tsx [hunk 1]
```diff
@@ -31,10 +31,10 @@ export function StakingData({ stakingData }: StakingDataProps) {
         <Panel>
             <Title title="Staking" />
             <div className="flex h-full w-full items-center gap-md p-md--rs">
-                <div className="w-1/2">
+                <div className="w-1/2 text-primary-80">
                     <LabelText
                         size={LabelTextSize.Large}
-                        label="Staked"
+                        label="Staked LAna"
```
Comment (panteleymonchuk) [inline apps/wallet-dashboard/components/staking-overview/StakingData.tsx]:
```suggestion
                        label="Staked"
```
?

**Review by panteleymonchuk — APPROVED** [review]
LGTM, just one minor comment

File: apps/core/src/utils/determineCountDownText.ts [hunk 2]
```diff
@@ -0,0 +1,16 @@
+// Copyright (c) Mysten Labs, Inc.
```
Comment (evavirseda) [inline apps/core/src/utils/determineCountDownText.ts:1]: shouldn't this header look like this?
```
// Copyright (c) 2024 IOTA Stiftung
// SPDX-License-Identifier: Apache-2.0
```

**Review by evavirseda — CHANGES_REQUESTED** [review]
looks nice, just one detail :pray:

File: apps/core/src/utils/determineCountDownText.ts (hunk 2 above)
Comment (brancoder) [reply apps/core/src/utils/determineCountDownText.ts:1]: Yes it's a new file, but the code is modified Mysten code from another file so I think its ok keeping the modification headers

Comment (evavirseda) [reply apps/core/src/utils/determineCountDownText.ts:1]: ah, got it! all good then :rocket:

**Review by evavirseda — APPROVED** [review]
nice job :star:

File: apps/wallet/src/ui/app/staking/validators/ValidatorsCard.tsx [hunk 3]
```diff
@@ -124,28 +140,46 @@ validator to start earning rewards again."
                     </div>
                 ) : null}
                 <div className="w-full gap-2">
+                    hey
```
Comment (begonaalvarezd) [inline apps/wallet/src/ui/app/staking/validators/ValidatorsCard.tsx]: :trollface:

File: apps/wallet/src/ui/app/staking/validators/ValidatorsCard.tsx [hunk 4]
```diff
@@ -124,28 +140,46 @@ validator to start earning rewards again."
...
+                                                staked: delegation.stakedIotaId,
+                                            }).toString()}`,
+                                        )
+                                    }
                                 />
                             ))}
                 </div>
 
                 <div className="w-full gap-2">
+                    hey
```
Comment (begonaalvarezd) [inline apps/wallet/src/ui/app/staking/validators/ValidatorsCard.tsx]: :trollface:

**Review by begonaalvarezd — CHANGES_REQUESTED** [review]

**Review by begonaalvarezd — APPROVED** [review]

**Pull request merged**